src/styles/main.scss

```diff
@@ -205,7 +205,7 @@ div.article-container{
     @media (max-width: $screen-sm) {
       width: 100%;
     }
-    min-height: calc(100vh - 68px); // make sure content always fills available space (68px is header size)
+    min-height: calc(100vh - 70px); // make sure content always fills available space (70px is header size)
```

Comment:
Can we make the header size a variable? Then don't have to manually updated in each place